### ISSUE TITLE
Dropout during prediction phase

### ISSUE THREAD

**Issue by ProfessorHT** [keras-team/tf-keras#81]
Hello,

I have used "tf.keras.layers.Dropout(0.2)(x, training=True)" to turn on the dropout during the prediction phase when i trained my model. Is there any method to turn it off again during the prediction when loading the h5 saved model?

I'am using TF version: 2.10.

Thank you

**Comment by tilakrayal** [comment]
@ProfessorHT,
While setting `training=True` during inference is the common approach for MC Dropout, Please find the alternative method for the same

Could you please try to create a custom dropout layer that works as if **training=True** even during inference.

```
class MCDropout(keras.layers.Dropout):
    def call(self, inputs, training=None):
        return super().call(inputs, training=True)  

```

Also try to re-train the model with training=True specifically for inference, effectively using dropout behavior for prediction and try to implement the dropout manually during inference, randomly zeroing out units as required. Thank you!
